src/lib/schematics/form/files/__path__/__name@dasherize@if-flat__/__name@dasherize__.component.html

```diff
@@ -0,0 +1,102 @@
+<form [formGroup]="addressForm" novalidate>
+  <mat-card class="shipping-card">
+    <mat-card-header>
+      <mat-card-title>Shipping Information</mat-card-title>
+    </mat-card-header>
+    <mat-card-content>
+      <table cellspacing="0" class="full-width">
```

Comment:
This shouldn't be a `table` because it's not showing tabular data.

Reply:
The other examples used a table so I just carried that over here.

Reply:
Which other examples? User-facing ones?